.github/workflows/org-policy.yaml

```diff
@@ -0,0 +1,67 @@
+name: orgpolicy
```

Comment:
non-blocking: should we standardize the use of hyphen or no hyphen in org-policy?